bve/src/parse/mesh/instructions/tests/meshes/mod.rs

```diff
@@ -0,0 +1,2 @@
+pub const CUBE_SOURCE: &str = include_str!("cube.obj");
+// pub const CUBE_RESULT: &'static str = include_str!("cube.res.obj");
```

Comment:
d e l e t

Reply:
but delete this plz

Reply:
I did, that latest commit removed it, and went and struckthrough the readme parts that were invalid.